docs/coordinate_systems.md

```diff
@@ -0,0 +1,361 @@
+
+- [API and Coordinates Usage](#api-and-coordinates-usage)
+  * [Important Notes](#important-notes)
+- [Coordinates System Definitions](#coordinates-system-definitions)
+  * [Physical Coordinates - Any Arch](#physical-coordinates---any-arch)
+    + [Important Notes](#important-notes-1)
+    + [Example](#example)
+    + [Mapping relationship between NOC0/NOC1](#mapping-relationship-between-noc0-noc1)
+  * [Logical Coordinates - Any Arch](#logical-coordinates---any-arch)
+    + [Important Notes](#important-notes-2)
+    + [Example](#example-1)
+  * [Hardware Translated Coordinates - Only for Wormhole and beyond](#hardware-translated-coordinates---only-for-wormhole-and-beyond)
+    + [Important Notes](#important-notes-3)
+    + [Example](#example-2)
+  * [Virtualized coordinates (Inputs and outputs from UMD)](#virtualized-coordinates--inputs-and-outputs-from-umd-)
+    + [Important Notes](#important-notes-4)
+- [Harvesting and Coordinates](#harvesting-and-coordinates)
+  * [Harvesting and Virtualization Example (Wormhole)](#harvesting-and-virtualization-example--wormhole-)
+    + [Physical Coordinates](#physical-coordinates)
+    + [Translated Coordinates](#translated-coordinates)
+    + [Virtual Coordinates](#virtual-coordinates)
+  * [Harvesting and Virtualization Example (Grayskull)](#harvesting-and-virtualization-example--grayskull-)
+      - [Physical Coordinates](#physical-coordinates-1)
+      - [Equivalent Virtual/Translated Coordinates](#equivalent-virtual-translated-coordinates)
+  * [Important Notes about Harvesting](#important-notes-about-harvesting)
+    + [Additional Notes about the Translation Scheme in UMD](#additional-notes-about-the-translation-scheme-in-umd)
+
+
+# API and Coordinates Usage
+## Important Notes
+* UMD accepts Virtual Coordinates (these match Physical Coordinates on Grayskull)
+* Any application programming overlay hardware/streams must use Translated Coordinates on a Wormhole Device with translation tables enabled
+* Regular Physical Coordinates can be used on a Wormhole Device with Translation Tables disabled
+* Translation Tables are a prerequisite for Wormhole Harvesting
+* As part of product definition, translation tables are enabled on Wormhole devices regardless of their grid size
+
+# Coordinates System Definitions
+## Physical Coordinates - Any Arch
+These are the NOC coordinates that the hardware understands, there are two distinct variations for NOC0 and NOC1.  In hardware, each node is given an ID (which is different for each NOC) which can be used to identify this node.  In the SOC descriptor, physical coordinates are specified for NOC0. 
+
+### Important Notes 
+* The coordinates are **NOT** contiguous for tensix cores -- In the example shown, ethernet corresponds to physical coords `[*-6]`:  this row is not on in the grid of worker cores
+* The coordinates are statically assigned to each "node" regardless of harvesting
+* `grayskull` are listed in `[y-x]` while `wormhole_b0` are listed in `[x-y]`
```

Comment:
Wow is this really true? We parse them from yaml differently?

Reply:
Removed the docs, going to address it in a separate PR (#223) when I rewrite it properly. I will resolve this comment there